.github/workflows/build.yml

```diff
@@ -324,7 +324,7 @@ jobs:
 
       - name: Add hosts to /etc/hosts
         run: |
-          echo "127.0.0.1 auth.nexodus.local api.nexodus.local nexodus.local" | sudo tee -a /etc/hosts
+          echo "127.0.0.1 auth.try.nexodus.local api.try.nexodus.local try.nexodus.local" | sudo tee -a /etc/hosts
```

Comment:
I would also be fine changing this back. We only need the added subdomain for prod.

Reply:
yeah I thought of that, but just for consistency sake, i changed it locally as well. Hopefully it won't create confusion when it comes to configuration of deployments (locally and prod/qa).